apps/src/netsim/NetSimVisualization.js

```diff
@@ -258,6 +258,44 @@ NetSimVisualization.prototype.getElementByEntityID = function (elementType, enti
 };
 
 /**
+ * Gets the set of VizWires directly attached to the given VizNode on
+ * the local end for which there are also corresponding VizWires coming
+ * from the opposite end. Note that if the VizNode is a router, we
+ * consider all attached wires to be reciprocated.
+ * @param {NetSimVizSimulationNode} vizNode
+ * @returns {Array.<NetSimVizSimulationWire>} the attached wires
```

Comment:
This is correct; just FYI, we also consider `{NetSimVizSimulationWire[]}` acceptable syntax here ([source](http://usejsdoc.org/tags-type.html)).